Sources/Danger/GitHubDSL.swift

```diff
@@ -215,7 +215,7 @@ extension GitHub {
     /// A GitHub team.
     public struct Team: Decodable, Equatable {
         /// The UUID for the team.
-        public let id: Int // swiftlint:disable:this identifier_name
+        public let id: Int 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 218;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)